sr_ethercat_hand_config/launch/sr_lhand.launch

```diff
@@ -3,11 +3,11 @@
     <arg name="log_bag_prefix" value="sr_lhand"/>
   </include>
   <include file="$(find sr_robot_launch)/launch/srhand.launch">
-    <arg name="eth_port" value="eth2" />
-    <arg name="hand_serial" value="1050" />
+    <arg name="eth_port" value="enp4s0f1" />
+    <arg name="hand_serial" value="111" />
     <arg name="hand_id" value="lh" />
 
-    <arg name="robot_description" value="$(find sr_description)/robots/shadowhand_left_motor.urdf.xacro" />
+    <arg name="robot_description" value="$(find sr_description)/robots/shadowhand_left_motor_plus.urdf.xacro" />
```

Comment:
@beatrizleon @toliver  are you fine with this change?

Reply:
If all new hands are going to be Plus, then I agree with the change.

Reply:
I agree with @toliver that if all are going to be plus then yes. 
It can also be replaced with:
<arg name="hand_type" default="hand_e_plus"/>